addons/hr_timesheet/models/project.py

```diff
@@ -21,6 +21,7 @@
     'subtask_planned_hours',
     'timesheet_ids',
     'total_hours_spent',
+    'remaining_hours_percentage',
```

Comment:
Why is it needed? 🤔

Reply:
> Why is it needed? thinking

I checked it but its not used anywhere so I removed.thnaks